javascript/js-inputs-and-strings_change-box-appearance/js/index.js

```diff
@@ -16,3 +16,22 @@ For further information check MDN:
 - about CSS declarations with "style": https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style
 
 */
+
+const inputColor = document.querySelector('[data-js="input-color"]')
+const inputRadius = document.querySelector('[data-js="input-radius"]')
+const inputRotation = document.querySelector('[data-js="input-rotation"]')
+const boxElement = document.querySelector('[data-js="box"]')
+
+inputColor.addEventListener("input", () => {
+    boxElement.style.backgroundColor = `hsl(${inputColor.value} 66% 70%)`
```

Comment:
did the same, Sven claimed that the hsl(..., ..., ...) would have to be divided through commas but I think it is not true